packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -116,6 +116,24 @@ export class Loan {
   async listCollateralTokens (): Promise<CollateralTokensData> {
     return await this.client.call('listcollateraltokens', [], 'bignumber')
   }
+
+  /**
+   * Creates (and submits to local node and network) a token for a price feed set in collateral token.
+   *
+   * @param {SetLoanToken} loanToken
+   * @param {string} loanToken.symbol Token's symbol (unique), no longer than 8
+   * @param {string} loanToken.name Token's name, no longer than 128
+   * @param {string} loanToken.priceFeedId Txid of oracle feeding the price
+   * @param {boolean} [loanToken.mintable] Token's 'Mintable' property, default=true
+   * @param {BigNumber} [loanToken.interest] Interest rate, default=0
```

Comment:
```suggestion
   * @param {BigNumber} [loanToken.interest=0] Interest rate, default=0
```

Reply:
Actually I don't set interest default to 0 in javascript

Reply:
should still update the docs since these APIs represent `defid` side functionality.
refer https://github.com/DeFiCh/jellyfish/blob/ee7767ccfb37128c4e4a9a52ac5a9a37e87762be/packages/jellyfish-api-core/src/category/icxorderbook.ts#L25

Reply:
I will apply webster suggestion and explicitly set it to `0`, this allows us to greatly control the optionality of the RPC instead of reliance on defid. This makes us resilient to shock changes from upstream.

Reply:
better.

Reply:
I think once we set the default on our side we don't ever change it again, that's our guarantee. If we need to change something, we create another method forit.

Reply:
We might have some refactoring then. https://github.com/DeFiCh/jellyfish/issues/649

Reply:
Fixed